src/script/conversation/EventMapper.ts

```diff
@@ -295,6 +297,11 @@ export class EventMapper {
         break;
       }
 
+      case ClientEvent.CONVERSATION.FEDERATION_STOP: {
+        messageEntity = this._mapEventFederationStop(event);
+        break;
+      }
+
```

Comment:
don't we need to handle the connectionRemove event (and display it) as well?

Reply:
this is the event that we are creating on the client side for the system message, connection removed is handled here:
https://github.com/wireapp/wire-webapp/pull/15583/files#diff-8424911ba55dc95ddd7ee35e77e651a8894faa0c33cd485cbbf62f6537fe3d24R327-R329

Reply:
but there is no system message for connectionRemoved event?